borgmatic/hooks/mysql.py

```diff
@@ -153,31 +164,31 @@ def dump_data_sources(databases, config, log_prefix, dry_run):
             for dump_name in dump_database_names:
                 renamed_database = copy.copy(database)
                 renamed_database['name'] = dump_name
-                processes.append(
-                    execute_dump_command(
-                        renamed_database,
-                        log_prefix,
-                        dump_path,
-                        (dump_name,),
-                        extra_environment,
-                        dry_run,
-                        dry_run_label,
-                    )
-                )
-        else:
-            processes.append(
-                execute_dump_command(
-                    database,
+                result = execute_dump_command(
+                    renamed_database,
                     log_prefix,
                     dump_path,
-                    dump_database_names,
+                    (dump_name,),
```

Comment:
I'm not quite following this changed and the earlier deletions. Don't you still need some of that logic if the database name is `all`?